### PR TITLE
[atlantis-experimental] Add cloudposse/atlantis

### PR DESCRIPTION
**Pull request by osterman**
## what
* Add the cloudposse/atlantis distribution of atlantis

## why
* This is what we're currently supporting

### CODE REVIEW

**Comment by aknysh** [comment]
> @joshmyers, @aknysh any thoughts on naming convention here? what should we call our own fork?
> 
> I don't want to add `-cloudposse` because installation would look like this `apk add atlantis-cloudposse@cloudposse`

hmm, good point.
on the one hand, people got used to the name (it's like `googling` now).
on the other hand, we need to distinguish our fork that has different sets of features.

maybe something like this:

`atlantisX` (sounds familiar too :)
`atlantics`
`atlantic`

File: vendor/atlantis-experimental/VERSION [hunk 1]
```diff
@@ -0,0 +1 @@
+0.5.3
```
Comment (aknysh) [inline vendor/atlantis-experimental/VERSION]:
```suggestion
0.6.0
```

Add latest version? (maybe not tested yet)

**Comment by osterman** [comment]
Going to create a new apk distribution (e.g. `experimental` and put it there). Blocked on time =)

**Comment by osterman** [comment]
Closed by #245

**Pull request closed without merging**